python/CMakeLists.txt

```diff
@@ -1,16 +1,23 @@
+set(PROJECT_PYTHON_SOURCE_DIR ${PROJECT_SOURCE_DIR}/python)
 set(GTSAM_PYTHON_BUILD_DIRECTORY ${PROJECT_BINARY_DIR}/python)
 
 if (NOT GTSAM_BUILD_PYTHON)
     return()
 endif()
 
+# Install development dependencies to build wrapper
```

Comment:
We really should not do this in CMakeLists, should be in the installation instructions instead

Reply:
I agree, though I was wondering about this since @dellaert prefers things being automated. I'll remove this for now and update the README, and let Frank leave a comment when he gets a chance. We can always add it in a follow-up PR.